src/org/joml/Vector2d.java

```diff
@@ -676,32 +669,77 @@ public double lengthSquared() {
      * @return the length squared of the given vector
      *
      * @author F. Neurath
+     *
+     * @deprecated method name has a typo, use {@link #lengthSquared(double, double)} instead
      */
     public static double lengthSqared(double x, double y) {
         return x * x + y * y;
     }
 
+    /* (non-Javadoc)
+     * @see org.joml.Vector2dc#length()
+     */
+    public double length() {
+        return Math.sqrt(lengthSquared());
+    }
+
+    /**
+     * Get the length of a 2-dimensional double-precision vector.
+     * Addresses <a href="https://github.com/JOML-CI/JOML/issues/131">Issue #131</a>
```

Comment:
Please remove the Addresses <issue> line. This is usually of little use to JOML users. And if needed one can use git diffs/annotates to see which method was introduced in which commit.